examples/experimental/trial/unets_tf_keras/const.yaml

```diff
@@ -0,0 +1,27 @@
+description: unets_tf_keras_const
+data:
+  BUFFER_SIZE: 1000
+  data_dir: '/tmp/data'
+  data_file: mobilenet_v2_weights_tf_dim_ordering_tf_kernels_1.0_128_no_top.h5
+
+hyperparameters:
+  learning_rate: 1e-4
+  learning_rate_decay: 1e-6
+  layer1_dense_size: 16
+  global_batch_size: 64
+  OUTPUT_CHANNELS: 3
+
+searcher:
+  name: single
+  metric: val_acc
+  smaller_is_better: false
+  max_steps: 230 # There are ~7400 examples (200 examples/class * 37 classes) in the training set
+                   # We run for 20 epochs as per the Tensorflow UNet tutorial. We get 230 by:
+                   # 7400 / global_batch_size / (1 batch per step) * 20 epochs
+
+min_validation_period: 10
+max_restarts: 0
+entrypoint: model_def:UNetsTrial
+environment:
+  image:
```

Comment:
Is the image field supposed to be blank -- can we directly set the environment instead of having the gpu field as well?

Reply:
This is how our docs demonstrate it: https://docs.determined.ai/latest/how-to/custom-env.html#tf2-environment, so I'm going to mimic them.